src/gameComponents/Ship/ProtoShip.java

```diff
@@ -0,0 +1,21 @@
+package gameComponents.Ship;
+
+import BBDGameLibrary.Geometry2d.BBDPoint;
+
+/**
+ * A class to serve as a precursor to a proper ship class.  Primarily serves as a testing
+ * bed for building new objects and the like.
+ */
+public class ProtoShip {
+    private enum Size {SMALL, MEDIUM, LARGE}
+
+    private BBDPoint currentLocation = new BBDPoint(0,0);
```

Comment:
does bbd point use vectors?

Reply:
Maybe?  Mostly it comes packaged with a lot of utility type functions for point related comparisons and modifications and plays nice with more complex geometry functions like distance measuring, is Point on Segment etc.

Very rarely are we going to need to know where the center is outside of render time.

Reply:
fair enough